adafruit_matrixportal/matrixportal.py

```diff
@@ -149,7 +149,7 @@ def add_text(
         """
         if text_font:
             if text_font is terminalio.FONT:
-                self._text_font = text_font
+                self._text_font = terminalio.FONT
```

Comment:
Why change this?

Reply:
It didn't really need to be changed, I just thought it was more readable.